version/current_test.go

```diff
@@ -39,7 +39,7 @@ func (*CurrentSuite) TestCurrentSeries(c *gc.C) {
 			if s != "n/a" {
 				// There is no lsb_release command on CentOS.
 				if currentOS == os.CentOS {
-					c.Check(s, gc.Matches, `centos7|centos8`)
+					c.Check(s, gc.Matches, `centos7|centos8|centos9`)
```

Comment:
can we match on `centos\d+` or something to make it more robust?

Reply:
😆